src/components/select/select.tsx

```diff
@@ -194,7 +194,7 @@ export class Select {
                 required={this.required}
                 invalid={this.invalid}
                 label={this.label}
-                helperText={this.helperText}
+                helperText={this.menuOpen ? this.helperText : ''}
```

Comment:
this doesn't seem to be necessary, or correct. Can you please explain what you want to achieve here?

Reply:
i see… 🤔 
But isn't that just the symptom? It should not be showing up automatically, if we solve the focusability with keyboard. In other words, if we disable the trigger for real (as I explained in the other comment).